src/util.ts

```diff
@@ -23,3 +24,8 @@ export enum HeadlessPromptSpecialVersion {
   LATEST = 'latest',
   DANGEROUSLY_USE_UNDEPLOYED = 'dangerously-use-undeployed',
 }
+
+export const AUTOBLOCKS_HEADERS = {
+  'Content-Type': 'application/json',
+  'X-Autoblocks-SDK': `javascript-${packageJson.version}`,
+};
```

Comment:
Wherever we are consuming this downstream - we can infer the SDK version is less than whatever version this release ends up being if the Autoblocks SDK header is not present - not sure if we want to keep track of that but maybe useful